<launch>

  <arg name="namespace" default="docker"/>
  <arg name="debug" default="false"/>

  <!-- CHANGE PNG_LEVEL TO 1 -->
  <!-- https://github.com/knorth55/zdepth_image_transport -->
  <!-- https://github.com/knorth55/zdepth_image_transport -->
  <include file="$(find detic_ros)/launch/sample_detection.launch">
    <arg name="input_image" value="/realsense_torso/color/image_rect_color"/>
    <arg name="input_depth" value="/realsense_torso/aligned_depth_to_color/image_raw"/>
    <arg name="input_camera_info" value="/realsense_torso/color/camera_info"/>
    <arg name="robot_frame" value="base"/>
    <arg name="vocabulary" value="custom"/>
    <arg name="custom_vocabulary" value="baseball,ball,box,package,toy,bottle,cup"/>
    <arg name="confidence_threshold" value="0.2"/>
    <arg name="debug" value="$(arg debug)"/>
    <arg name="namespace" value="$(arg namespace)"/>
  </include>

  <!-- <group ns='$(arg namespace)'> -->
  <!--   <node name="detic_card_filter" -->
  <!--         pkg="detic_ros" type="filter_posearray.py" -->
  <!--         args="detic_segmentor/output/centroid detic_segmentor/output/card/table/centroid -->
  <!--               detic_euclidean_clustering/output detic_segmentor/output/card/table/indices"/> -->
  <!-- </group> -->

</launch>
